packages/valory/services/trader/service.yaml

```diff
@@ -143,6 +143,7 @@ type: skill
         max_attempts: ${MAX_ATTEMPTS:int:10}
         max_healthcheck: ${MAX_HEALTHCHECK:int:120}
         multisend_address: ${MULTISEND_ADDRESS:str:0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761}
+        multisend_batch_size: ${MULTISEND_BATCH_SIZE:int:5}
```

Comment:
Please set the batch size to 1 or set the env var in the trader quickstart to 1 and link the PR here.